docs/data/sdks/android/index.md

```diff
@@ -155,6 +155,24 @@ Accurate session tracking requires that you enable `enableForegroundTracking(ge
     val sameClient = Amplitude.getInstance("Andy_Client")
     ```
 
+#### Configuration
+
+???config "Configuration Options"
+    | <div class="big-column">Name</div>  | Description | Default Value |
+    | --- | --- | --- |
+    | `eventUploadPeriodMillis` | The amount of time SDK will attempt to upload the unsent events to the server or reach `eventUploadThreshold` threshold. | `30000` |
+    | `eventUploadThreshold` | SDK will attempt to upload once unsent event count exceeds the event upload threshold or reach `eventUploadPeriodMillis` interval.  | `30` |
+    | `serverUrl` | The server url events upload to. | `https://api2.amplitude.com/` |
+    | `identifyBatchIntervalMillis` | The amount of time SDK will attempt to batch intercepted identify events. | `30000` |
+    | `sessionTimeoutMillis` | The amount of time for session timeout if enable foreground tracking. | `1800000` |
+    | `eventUploadMaxBatchSize` | The maximum number of events sent with each upload request. | `50` |
+    | `minTimeBetweenSessionsMillis` | The amount of time for session timeout if disable foreground tracking. | `300000` |
+    | `eventMaxCount` | The maximum number of unsent events to keep on the device. | `1000` |
+    | `flushEventsOnClose` | Flushing of unsent events on app close. | `true` |
+    | `trackingSessionEvents` | Flushing of unsent events on app close. | `false` |
+    | `useDynamicConfig` |  Find the best server url automatically based on users' geo location. | `false` |
+    | `optOut` | Opt the user out of tracking. | `false` |
```

Comment:
Not sure the best order here. But maybe we should group by functionality - event upload options, session options, server options, etc

Reply:
Yeah, order is another part I would like to check. I will update it to group by functionality and give it a try.

Reply:
Just update the order based on the suggestion.